### DISCUSSION TITLE
[Feature request] Explore: Custom log levels

### DISCUSSION THREAD

**Discussion by kycfeel** [grafana/grafana#45680] [Observability]
**What would you like to be added**:
Custom log level (labeling) support would be very useful for users who got their own level on the log lines. For example, my application got `verbose` as a log level, but the grafana explore is marking it as `unknown`. 

**Why is this needed**:
Depends on the applications and user's personal needs, the log level could be different from the grafana's hard coded ones. Custom log level support would give users more insight and flexibility for tracking down their applications. No one likes to see `unknown` label on their dashboard :)

**Comment by ivanahuckova** [comment]
Also as mentioned here https://github.com/grafana/grafana/issues/24329#issuecomment-725665627, if we are going to support custom log levels and add configuration for this, we should also add option to disable Grafana's log parsing to get logLevel for users who don't want it.

**Comment by KornKalle** [comment]
Are there any news on this? 
I would love to see this as we are sending a relatively huge amount of logs to loki from various stacks and in a framework we currently started working with, the default value for info logs is just "log" instead of "info".
It would be awesome to make it easier to handle this and similar deviating default values through loki instead of overwriting the default behaviour on application side.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82945. Feel free to continue the discussion around this there. Thank you!
